src/main/java/mate/jdbc/service/DriverService.java

```diff
@@ -0,0 +1,18 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
```

Comment:
On the task, the method should be called  -  Driver get(Long id);

Reply:
Okay, got it